docs/getting_started/index.rst

```diff
@@ -466,18 +465,17 @@ Yielding the output:
 
 .. literalinclude:: xnvme_win_io_async_read_iocp_th.out
    :language: bash
-   :lines: 1-12
 
 Async I/O via ``io_ring``
-~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**xNVMe** utilizes the Windows **io_ring** interface, its support for
+**xNVMe** utilizes the Windows **ioring** interface, its support for
```

Comment:
Is this `ioring` different from the  `io_ring` in the line below

Reply:
fixed.